build/_npm.shade

```diff
@@ -5,12 +5,12 @@ default npmDir = '${currentDir}'
 node-install once='installNode'
 
 var npmFile = '${ Path.Combine(nodeDir, "npm.cmd") }'
-var npmCmd = '${ File.Exists(npmFile) ? npmFile : "npm" }'
+var npmCmd = '${ File.Exists(npmFile) ? ("\"" + npmFile + "\"") : "npm" }'
```

Comment:
no because we're in a C# code block; I'm just wrapping the expression that adds the quotes in parens to make the precedence more clear